### PR TITLE
Kenku updates

### PR DESCRIPTION
**Pull request by ocelot134269**

### CODE REVIEW

File: areas/startlocs.cpp [hunk 1]
```diff
@@ -163,6 +164,11 @@ bool startingChoices(Player *player, std::string str, char *location, bool choos
 
         options.emplace_back("kataran");
 
+    } else if (race == KENKU) {
+
+        options.emplace_back("Tokoki Peak");
```
Comment (mitchej123) [inline areas/startlocs.cpp:169]: pretty sure this is the area name, not display name

File: include/flags.hpp [hunk 2]
```diff
@@ -711,7 +711,7 @@
 // free                             114
 // free                             115
 // free                             116
-// free                             117
+#define O_SEL_KENKU                 117      // race selective: kenku
```
Comment (mitchej123) [inline include/flags.hpp:714]: is there nothing closer to the other ones? I recall it being some math to figure out which flag it was.

File: include/flags.hpp [hunk 3]
```diff
@@ -861,6 +861,7 @@
 #define X_LOOK_ONLY                 94      // Can only look through this exit (free scout)
 #define X_LOCK_AFTER_USAGE          95      // Exit re-locks after single usage
 #define X_NO_KNOCK_SPELL            96      // Exit immune to knock spell
-#define MAX_EXIT_FLAGS              97      // Incriment when adding... check structs.h for max
+#define X_SEL_KENKU                 97      // race selective: kenku
```
Comment (mitchej123) [inline include/flags.hpp:864]: is there nothing closer to the other ones? I recall it being some math to figure out which flag it was.

File: areas/startlocs.cpp (hunk 1 above)
Comment (ocelot134269) [reply areas/startlocs.cpp:169]: I thought so too at first, but when i had it as kenku, the area option on char create was "[A] Kenku"...and when i put it as Tokoki Peak, it was [A] Tokoki Peak.

File: include/flags.hpp (hunk 2 above)
Comment (ocelot134269) [reply include/flags.hpp:714]: It checks the flags individually. No math.

File: include/flags.hpp (hunk 3 above)
Comment (ocelot134269) [reply include/flags.hpp:864]: It checks the flags individually. No math.

**Review by mitchej123 — APPROVED** [review]

**Pull request merged**